src/org/camunda/latera/bss/http/HTTPRestProcessor.groovy

```diff
@@ -141,22 +144,29 @@ class HTTPRestProcessor {
       logger.debug("--------------------")
     }
 
-    def result = httpClient."${method}" {
-      response.success responseBlock(false, supressResponseBody)
-      response.failure responseBlock(true,  supressResponseBody)
-
-      if (params.path) {
-        request.uri.path = absolutePath(params.path.toString())
-        params.remove('path')
-      }
-      if (params.query) {
-        request.uri.query = params.query
-        params.remove('query')
-      }
-      params.each { k,v ->
-        request."${k}" = v
+    def result = [:]
```

Comment:
It's better to left here just:
```groovy
def result
```
without Map value, because it can cause misunderstanding about result type.